### PR TITLE
Minor misc fixes to religion

### PR DESCRIPTION
**Pull request by ShiftyRail**
Brian Damag: Diavil on discord ran into problems with this one.
Changeling: was asked in-game.

### CODE REVIEW

**Comment by Diavill** [comment]
skellington 4 shiters

**Comment by DamianX** [comment]
Isn't the deity "Brian Damag"?

**Comment by ShiftyRail** [comment]
@DamianX You're right, I added an extra e, my bad
All keys must be in lowercase however

File: code/datums/religions.dm [hunk 1]
```diff
@@ -733,8 +733,8 @@
 
 /datum/religion/changeling
 	name = "The Religion" // A la "The Thing"
-	deity_name = "Proboscis"
-	bible_name = "The Hive"
+	deity_name = "The Hive"
+	bible_name = "Probicis"
```
Comment (DamianX) [inline code/datums/religions.dm]: prowhat?

**Comment by DamianX** [comment]
and what happened to brian damag now?

**Comment by ShiftyRail** [comment]
I blame GitHub

**Pull request merged**